src/components/BottomNavigationBot/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, memo } from 'react'
+import ModalContainer from '../ModalContainer'
+import './styles.scss'
+import botIcon from '../../assets/bot.svg'
+
+const BottomNavigationBot = () => {
+  const [showBot, setShowBot] = useState(false)
+  const openModal = () => setShowBot(true)
+  const closeModal = () => setShowBot(false)
+
+  return (
+    <div className="nav-bot">
+      <button type="button" className="button-image" onClick={openModal}>
```

Comment:
You could use the component ButtonContainer here, which already has keypress events and tabIndex implemented for accessability :)